File: from-1701-to-1800/1752-check-if-array-is-sorted-and-rotated/solution.go
func check(nums []int) bool {
    rotationPoints := 0
    n := len(nums)
    for i := 1; i < n; i ++ {
        if nums[i - 1] > nums[i] {
            rotationPoints += 1
        }
    }
    if nums[n - 1] > nums[0] {
        rotationPoints += 1
    }
    return rotationPoints <= 1
}

File: from-1701-to-1800/1752-check-if-array-is-sorted-and-rotated/solution.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        n = len(nums)
        rotation_points = 0
        for i in range(1, n):
            if nums[i - 1] > nums[i]:
                rotation_points += 1
                
        if nums[-1] > nums[0]:
            rotation_points += 1
        
        return rotation_points <= 1